_episodes/06-practical-recommendations.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Practical recommendations for team work"
+teaching: 5
+exercises: 20
+questions:
+- "What are practical tips for doing team work?"
+- "What soft skills are valuable when working in teams?"
+- "How can I apply what I learned in my daily work?"
+objectives:
+- "List a few practical recommendations for doing teamwork"
+- "Know which soft skills make great teams"
+keypoints:
+- "Learn from other people's experiences working in teams."
+- "Certain soft skills are vital for teamwork."
+---
+# Practical recommendations
+> ## Discussion: Your practical tips.
+> Based on your own experience, what would be your top-3 practical tips that
+> you would like to give someone that is new to working in a team?
+> This can be based on experience with formal teams, in pairs, or any other form of collaborative work.
+{: .discussion}
+
+Our tips:
+- **Any mutation to the team makes a new team.** Whenever a team member leaves or is added to the team,
+you should treat this as the start of a new team.
+- **Teams need a coach.** Self-organization of teams doesn't happen automatically.
+It can be very beneficial to have someone in or close to the team that can coach the
+team in the right direction.
+- **Enjoy.** Value joy in team meetings and interactions.
+Working together in an enthusiastic group of people makes all the rest simple.
+- **Invest in soft skills.** Certain soft skills are vital for teamwork, see below.
+
+# Soft skills
+Teams stand or fall with good soft skills of team members.
+
+> ## Discussion: Soft skills.
+> Which soft skills do you think are important to have when doing teamwork.
+> For example, think about applying the agile frameworks that we learned about,
+> what soft skills are important?
+{: .discussion}
+
+These are some important soft skills needed in teams:
+### Giving good feedback
+Learning from each other is one of the benefits of team work.
+For this, giving good feedback is vital.
+#### How to give good feedback?
+* Make it specific
+* Give feedback from your perspective. Feedback is always subjective.
+* Don't wait too long with giving feedback.
+* Stay positive. A good rule is to always start with something positive.
+* Prepare your comments.
+#### How to receive feedback?
+* Don't feel attacked. The person giving you feedback is trying to help you.
+* Ask for clarification if the feedback is not specific enough.
```

Comment:
It might be good again to point to further reading (or watching) about giving feedback. 
```suggestion
* Ask for clarification if the feedback is not specific enough.

If you want to learn more about giving feedback in a way that is pleasant for others, we recommend that you take a course dedicated to this topic with your whole team. For a first impression, here is a short [guide about giving constructive feedback](https://www.workpatterns.com/guides/constructive-feedback). 
```